src/node.cc

```diff
@@ -3265,7 +3273,8 @@ static void ParseArgs(int* argc,
   new_argv[0] = argv[0];
 
   unsigned int index = 1;
-  while (index < nargs && argv[index][0] == '-') {
+  bool short_circuit = false;
```

Comment:
I guess there isn't currently.  It wasn't entirely clear to me it was for passing extra arguments verbatim.